docs/DeveloperGuide.md

```diff
@@ -453,23 +460,125 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Adding a friend
+1. Adding a friend to gitGud
 
-1. Deleting a person while all persons are being shown
+    1. Prerequisites: List all friends using the `friend --list` command. There should not be a friend with 
+       `FRIEND_ID` Draco already stored in gitGud.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+    2. Test case: `friend --add Draco --name Marcus`<br>
+       Expected: Friend with `FRIEND_ID` Draco is added. gitGud states that `FRIEND_ID` is added.
+       
+    3. Test case: `friend --add MrFeely`<br>
+       Expected: Friend with `FRIEND_ID` MrFeely is added. gitGud states that `FRIEND_ID` is added.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+    4. Other incorrect add commands to try: `friend --name Marcus`, `friend --add`, `friend --name`, ...`
+       Expected: Similar to previous.
```

Comment:
Not sure if incorrect commands should be similar to adding a friend with FRIEND_ID MrFeely